TensorFlow/workspace/training_demo/training/ssd_inception_v2_coco.config

```diff
@@ -139,8 +139,8 @@ train_config: {
       learning_rate: {
         exponential_decay_learning_rate {
           initial_learning_rate: 0.01
```

Comment:
I'd suggest a higher lr, around 0.0001

Reply:
0.0001 is lower, not higher. It gives worse results in the beginning and in the end. I have tried. With higher/bigger LR we get faster convergence in the beginning, while we are far away from optimum